src/ResourceManager/Compute/Commands.Compute/Extension/AzureDiskEncryption/Scripts/AzureDiskEncryptionPreRequisiteSetup.ps1

```diff
@@ -63,10 +63,20 @@ $ErrorActionPreference = "Stop"
         $now = [System.DateTime]::Now;
         $oneYearFromNow = $now.AddYears(1);
         $aadClientSecret = [Guid]::NewGuid().ToString();
-        $secureAadClientSecret = ConvertTo-SecureString -String $aadClientSecret -AsPlainText -Force;
-
         Write-Host "Creating new AAD application ($aadAppName)";
-        $ADApp = New-AzureRmADApplication -DisplayName $aadAppName -HomePage $defaultHomePage -IdentifierUris $identifierUri  -StartDate $now -EndDate $oneYearFromNow -Password $secureAadClientSecret;
+
+        #New-AzureRmADApplication cmdlet expects SecureString as -Password parameter for versions >= 5.0.0. Earlier versions expect plain string.
```

Comment:
@SudhakaraReddyEvuri the build is failing because CredScan thinks "-Password parameter" is exposing a password. You can either remove this line or suppress the exception by adding the following above the line:

```
<#[SuppressMessage("Microsoft.Security", "CS002:SecretInNextLine", Justification="...")]#>
```

Reply:
@cormacpayne Fixed this by removing